core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -2143,11 +2144,17 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
   }
 
   // Verify that createTopics and alterConfigs fail with null values
-  @Test
-  def testNullConfigs(): Unit = {
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumName)
+  @ValueSource(strings = Array("zk", "kraft"))
+  def testNullConfigs(quorum: String): Unit = {
 
     def validateLogConfig(compressionType: String): Unit = {
-      val logConfig = zkClient.getLogConfigs(Set(topic), Collections.emptyMap[String, AnyRef])._1(topic)
+      val logConfig = if (isKRaftTest()) {
```

Comment:
Since we are consulting the metadata cache here, maybe it makes sense to call `ensureConsistentKRaftMetadata()`  here. Then we can get rid of the call below.

While we're at it, perhaps we can replace a few explicit calls to `TestUtils.ensureConsistentKRaftMetadata`?

Reply:
Yes, this can be more reasonable, and I also replaced some `TestUtils.ensureConsistentKRaftMetadata()` with `this.ensureConsistentKRaftMetadata()`.